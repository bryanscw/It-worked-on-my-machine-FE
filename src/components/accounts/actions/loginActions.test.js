import React from 'react'
import { render, cleanup } from '@testing-library/react'
import {fetchMe, authenticateLogin }from './loginActions.js';


afterEach(cleanup);

it('should take a snapshot', () => {
    const { asFragment } = render(<fetchMe />);
    expect(asFragment()).toMatchSnapshot();
});

it('should take a snapshot', () => {
    const { asFragment } = render(<authenticateLogin />);
    expect(asFragment()).toMatchSnapshot();
});

