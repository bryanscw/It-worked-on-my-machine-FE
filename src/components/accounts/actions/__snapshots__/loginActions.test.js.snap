// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <fetchme />
</DocumentFragment>
`;

exports[`should take a snapshot 2`] = `
<DocumentFragment>
  <authenticatelogin />
</DocumentFragment>
`;
